### DISCUSSION TITLE
Is it possible to add into [BaseUrl]/$(Build.BuildNumber) adding /index.html on the path? to display my report?

### DISCUSSION THREAD

**Discussion by Rogatto** [allure-framework/allure2#1448] [Questions & Support] [answered]
Is it possible to add into [BaseUrl]/$(Build.BuildNumber) adding /index.html on the path? to display my report on azure devops with the plugin  Allure Test Reports by Molecula?

**Comment by jamilhf** [comment]
@Rogatto hi were u able to find a work around for this plugin ? i am facing the same issue

**Comment by Rogatto** [comment]
> @Rogatto hi were u able to find a work around for this plugin ? i am facing the same issue

Hey @jamilhf 

I found a way to work correctly with Allure using Azure Devops Pipelines:

I didn't use this plugin anymore, there are a lot of problems using that in a CI/CD, so I have started to use a task as command line and execute the command line of Allure manually and native.

Before you must download the Allure command line in the agent machine. (you can use a nodeJS task to download the Allure with that)

At the end of the execution, I used this step to the report link address appears as a new tab of the pipeline (in my case the agent machines was Windows so I used powershell script):

- powershell: |
              $UrlFile = '$(System.DefaultWorkingDirectory)\My_Temp_Html_File.html'
              $UrlLink = '<a href="https://my hostallure/imagens/allure/$(allureReport)/$(Build.BuildNumber)/index.html">https://myhostallure/imagens/allure/$(allureReport)/$(Build.BuildNumber)/index.html</a>'
              Add-content -path $UrlFile -value $UrlLink
              Write-output "##vso[task.addattachment type=Distributedtask.Core.Summary;name=Integration Testing - Allure Report;]$UrlFile"
            displayName: 'Allure Link'

**Comment by jamilhf** [comment]
were u using your vm to host the url ?

**Comment by Rogatto** [comment]
> were u using your vm to host the url ?

@jamilhf 

You can easily use AWS S3 to put there the reports file generated (use the value of $(Build.BuildNumber) to create the folder) and access with http the report.

**Comment by jamilhf** [comment]
@Rogatto do u mind sharing a copy of yml file please

**Comment by sharafi86** [comment]
@Rogatto 
Would you mind helping us get the guide on how did you implement it to display the report from Azure DevOps, please?

**Comment by baev** [comment, marked as answer]
Official Allure extension for Azure Pipelines are now available https://marketplace.visualstudio.com/items?itemName=qameta.allure-azure-pipelines

History isn't yet supported, but we'll get there! Please submit feedback to our [discussions](https://github.com/orgs/allure-framework/discussions)
